src/main/scala/nl/knaw/dans/easy/stage/EasyStageDataset.scala

```diff
@@ -72,13 +73,22 @@ object EasyStageDataset {
     for {
       dataDir <- getDataDir
       _ <- mkdirSafe(s.sdoSetDir)
-      emdContent <- createDatasetSdo()
+      (emdContent,amdContent) <- createDatasetSdo()
       _ = log.info("Creating file and folder SDOs")
       _ <- createFileAndFolderSdos(dataDir, DATASET_SDO, emdContent.getEmdRights.getAccessCategory)
-    } yield ()
+    } yield (emdContent,amdContent)
   }
 
   def createFileAndFolderSdos(dir: File, parentSDO: String, rights: AccessCategory)(implicit s: Settings): Try[Unit] = {
+    val maybeSha1Map: Try[Map[String, String]] = Try {
```

Comment:
According to @rvanheest that library is quite complex to use and we don't have it on board in this component for any other purposes.